pkg/operator/controller/ingress/status.go

```diff
@@ -339,17 +420,8 @@ func computeDeploymentReplicasAllAvailableCondition(deployment *appsv1.Deploymen
 // conditions.
 func computeIngressDegradedCondition(conditions []operatorv1.OperatorCondition) (operatorv1.OperatorCondition, error) {
 	var requeueAfter time.Duration
```

Comment:
This declaration is redundant now and can be removed.

Reply:
Done.